repos/spack_repo/builtin/packages/fplll/package.py

```diff
@@ -29,6 +30,7 @@ class Fplll(AutotoolsPackage):
     version("5.1.0", sha256="58175c54cc92752576a64361c73e4ea7797fc18fb703b3f22c7570a09075486f")
     version("5.0.3", sha256="d2b11b7dcb26c30ac1aab9ff75aca9b3dd6e0b0b40c382af16017a717dfe05c2")
 
+    depends_on("c", type="build")
     depends_on("cxx", type="build")  # generated
```

Comment:
Can you confirm `cxx` is a dependency and, if so, remove `# generated` comment?

Reply:
Yes, it's definitely a dependency!  Without it, we get:

```
     102    checking whether g++ supports C++11 features with -std=c++11... no
     103    checking whether g++ supports C++11 features with +std=c++11... no
     104    checking whether g++ supports C++11 features with -h std=c++11... n
            o
     105    checking whether g++ supports C++11 features with -std=c++0x... no
     106    checking whether g++ supports C++11 features with +std=c++0x... no
     107    checking whether g++ supports C++11 features with -h std=c++0x... n
            o
  >> 108    configure: error: *** A compiler with support for C++11 language fe
            atures is required.
```

Comment removed